config/sync/core.extension.yml

```diff
@@ -7,7 +7,7 @@ module:
   dpl_login: 0
   dpl_mapp: 0
   dpl_react: 0
-  dpl_search: 0
+  dpl_react_apps: 0
```

Comment:
Consider whether this is the right approach.

This will effectively put all React applications into a single module, `dpl_react_apps` but what will then happen with any Drupal code related to searching e.g. configuration forms, server side code etc. Does this go in another `dpl_search` module or in `dpl_react_apps`?

As I see it we already have one supporting "tech" module - `dpl_react`. I suggest that we from here branch into domain related modules e.g. `dpl_search` which contains all code related to handling this domain.

Reply:
I leave it like this for now.
Would make sense to continue this discussion with ITK